__tests__/nc-news.test.js

```diff
@@ -25,6 +25,31 @@ describe('GET /api/topics', () => {
   });
 });
 
+describe('GET /api/articles/:article_id', () => {
+  it('200: return article by given Id', async () => {
+    const {body} = await request(app).get('/api/articles/1').expect(200);
+
+    expect(body.article).toBeInstanceOf(Object);
+    expect(body.article).toMatchObject({
+      author: expect.any(String),
+      title: expect.any(String),
+      article_id: expect.any(Number),
+      body: expect.any(String),
+      topic: expect.any(String),
+      created_at: expect.any(String),
+      votes: expect.any(Number),
+    });
+
+    expect(body.article.author).toBe('jonny');
+  });
+
+  it('404: return error if article does not exist on db', async () => {
+    const {body} = await request(app).get('/api/articles/20000').expect(400);
```

Comment:
Would this be a 400 or a 404? 
This id could exist in the future so I would say the input doesn't exist rather than a bad request e.g /api/articles/notAnId